meta.go

```diff
@@ -82,27 +160,72 @@ func getMeta(key string, metaSpace string, metaFile string, output io.Writer, js
 	decoder.UseNumber()
 	err = decoder.Decode(&metaInterface)
 	if err != nil {
-		return err
+		return "", err
 	}
 
 	_, result := fetchMetaValue(key, metaInterface)
 
 	switch result.(type) {
 	case map[string]interface{}, []interface{}:
 		resultJSON, _ := json.Marshal(result)
-		_, err = fprintf(output, "%v", string(resultJSON))
+		return fmt.Sprintf("%v", string(resultJSON)), nil
 	case nil:
-		_, err = fprintf(output, "null")
+		return "null", nil
 	default:
-		if jsonValue {
+		if m.JsonValue {
 			resultJSON, _ := json.Marshal(result)
-			_, err = fprintf(output, "%v", string(resultJSON))
+			return fmt.Sprintf("%v", string(resultJSON)), nil
```

Comment:
`golint` regarding the next line - I translate for clarity 😄
`meta.go:178:10: 'if' block ends with a return statement, so drop the 'else' keyword and outdent its block`

Reply:
Done